src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -941,24 +963,22 @@ private void discoverModCompat()
     }
 
     /**
-     * Gets all the possible flowers for the beekeeper.
-     * 
-     * @return a set of the flowers.
+     * Discover all the possible flowers for the beekeeper.
+     *
      */
-    public static Set<ItemStorage> getAllBeekeeperFlowers()
+    private void discoverFlowers()
     {
-        Set<ItemStorage> flowers = new HashSet<>();
-
-        for (final Item item : ForgeRegistries.ITEMS.tags().getTag(ItemTags.FLOWERS))
+        if (flowers.isEmpty())
         {
-            final NonNullList<ItemStack> list = NonNullList.create();
-            item.fillItemCategory(CreativeModeTab.TAB_SEARCH, list);
-            for (final ItemStack stack : list)
+            for (final Item item : ForgeRegistries.ITEMS.tags().getTag(ItemTags.FLOWERS))
```

Comment:
could you use the `allItems` List instead of refilling a creative tab here? since we already do that for all items

Reply:
This is still more efficient, as this just uses the sublist of items that are already tagged. Not ALL